storage_service/locations/api/resources.py

```diff
@@ -73,7 +73,7 @@
 def _is_relative_path(path1, path2):
     """Ensure path2 is relative to path1"""
     try:
-        Path(path2).resolve().relative_to(path1)
+        Path(path2).resolve().relative_to(Path(path1).resolve())
```

Comment:
`resolve` may raise other exceptions, do we need to catch them?

Reply:
I've added `RuntimeError` to catch symbolic link loops.